scripts/build2.sh

```diff
@@ -0,0 +1,49 @@
+#! /usr/bin/env bash
```

Comment:
I'm not sure the benefit of adding this script, it's just recreating what the other scripts already do, right?

Reply:
scripts/build1.sh -> clones golang-fips and run full-initialize-repo.sh

scripts/build2.sh -> clone golang binaries and apply patches one by one. 

In cases when working with downstream distros, users already have a go source tar e.g. centos go tar file (different from upstream vanilla go source that build1.sh uses), and they just care about applying patches on them, one by one. In those cases, second script helps.  Atleast that was my intention of adding the second script. I had to use the second one when building golang-fips on RH golang source imported from OpenELA.

Reply:
I see why someone might find these scripts useful, but they appear rather configuration and workflow specific.  Whereas the existing builds scripts are workflow agnostic and already capable of producing a Go toolchain with the same configuration.  I would suggest to instead that we document a more generic `disconnected` build approach using the patches, and avoid adding additional scripts.